src/Compilers/Core/CodeAnalysisTest/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -951,21 +951,11 @@ public sealed override void Initialize(AnalysisContext context)
                  (operationContext) =>
                  {
                      IEventAssignmentExpression eventAssignment = (IEventAssignmentExpression)operationContext.Operation;
-                     if (eventAssignment.Event.Name == "Mumble")
-                     {
-                         operationContext.ReportDiagnostic(Diagnostic.Create(eventAssignment.Adds? HandlerAddedDescriptor : HandlerRemovedDescriptor, operationContext.Operation.Syntax.GetLocation()));
-                     }
+                     operationContext.ReportDiagnostic(Diagnostic.Create(eventAssignment.Adds? HandlerAddedDescriptor : HandlerRemovedDescriptor, operationContext.Operation.Syntax.GetLocation()));
                  },
                  OperationKind.EventAssignmentExpression);
 
             context.RegisterOperationAction(
-                (operationContext) =>
-                {
-                    operationContext.ReportDiagnostic(Diagnostic.Create(EventReferenceDescriptor, operationContext.Operation.Syntax.GetLocation()));
-                },
-                OperationKind.EventAssignmentExpression);
```

Comment:
Confusing, perhaps, but are there other tests that more meaningfully cover EventAssignmentExpression? Your lambda test above looks like maybe it does.